include/openssl/digest.h

```diff
@@ -300,6 +300,11 @@ OPENSSL_EXPORT void EVP_MD_CTX_set_flags(EVP_MD_CTX *ctx, int flags);
 // EVP_MD_nid calls |EVP_MD_type|.
 OPENSSL_EXPORT int EVP_MD_nid(const EVP_MD *md);
 
+// |EVP_MD_CTX_set_pkey_ctx| sets a |EVP_PKEY_CTX| value to |EVP_MD_CTX|'s
```

Comment:
The `EVP_PKEY_CTX` object needs to have been initialised with an operation before associating it with the `EVP_MD_CTX` object.